### ISSUE TITLE
Support bump a part *to* a specified valid value

### ISSUE THREAD

**Issue by gzm55** [c4urself/bump2version#278]
Such as a _release_ part, has three values: dev, rc, prod. with this feature, we can do a dev-> prod bump, skipping the rc phase.

**Comment by florisla** [comment]
Hi, I belive this is not possible currently.

Also, note that bump2version is no longer maintained.

Please switch to the more modern [bump-my-version](https://github.com/callowayproject/bump-my-version).

**Comment by gzm55** [comment]
> Hi, I belive this is not possible currently.
> 
> Also, note that bump2version is no longer maintained.
> 
> Please switch to the more modern [bump-my-version](https://github.com/callowayproject/bump-my-version).

@florisla  is this feature supported in bump-my-version ?

**Comment by florisla** [comment]
I don't think it is.  But there's a chance that they would consider adding such a feature.
